.github/workflows/destroy-environment.yml

```
@@ -0,0 +1,54 @@
+name: Destroy Environment
+run-name: Destroying environments starting with ${{ github.event.inputs.prefix }} by @${{ github.actor }}
```

maybe something shorter?
```suggestion
run-name: Destroying ${{ github.event.inputs.prefix }}* by @${{ github.actor }}
```